lib/static.js

```diff
@@ -114,14 +114,14 @@ exports.text = function(elems) {
   }
 
   var ret = '',
-      len = elems.length,
+      q = elems.slice(),
       elem;
 
-  for (var i = 0; i < len; i++) {
-    elem = elems[i];
+  for (var i = 0; i < q.length; i++) {
+    elem = q[i];
     if (elem.type === 'text') ret += elem.data;
     else if (elem.children && elem.type !== 'comment') {
-      ret += exports.text(elem.children);
+      Array.prototype.push.apply(q, elem.children);
```

Comment:
This changes the behaviour and results in ordering errors. I'm not sure why it passed on Travis, and Travis won't allow me to view the build right now or sign in. I'll try checking again later.

It fails for me locally with:

```
  1) cheerio .text (cheerio object) : should return the text contents of the specified elements:
     Error: expected 'This is .content' to equal 'This is content.'
      at Assertion.assert (node_modules/expect.js/index.js:96:13)
      at Assertion.be.Assertion.equal (node_modules/expect.js/index.js:216:10)
      at Context.<anonymous> (test/api/utils.js:41:33)

  2) cheerio .text (cheerio object) : should include text contents of children recursively:
     Error: expected 'This is  text.a child with  followed by  and some finalanother child and  not a commentone last child' to equal 'This is a child with another child and  not a comment followed by one last child and some final text.'
      at Assertion.assert (node_modules/expect.js/index.js:96:13)
      at Assertion.be.Assertion.equal (node_modules/expect.js/index.js:216:10)
      at Context.<anonymous> (test/api/utils.js:51:33)

  3) cheerio .text () : should return the rendered text content of the root:
     Error: expected 'This is  text.a child with  followed by  and some finalanother child and  not a commentone last child' to equal 'This is a child with another child and  not a comment followed by one last child and some final text.'
      at Assertion.assert (node_modules/expect.js/index.js:96:13)
      at Assertion.be.Assertion.equal (node_modules/expect.js/index.js:216:10)
      at Context.<anonymous> (test/api/utils.js:56:27)
```

You would need to insert the children at the current index using `splice` or something.